src/pages/zh/smart-contracts/genesis/sdk/javascript.md

```
@@ -0,0 +1,379 @@
+|-------|-------|
+| `insufficient funds` | SOL 不足以支付费用 |
+| `already initialized` | Genesis Account 已存在 |
+| `already finalized` | 最终化后无法修改 |
+| `deposit period not active` | 不在存款窗口内 |
+| `claim period not active` | 不在领取窗口内 |
+
+---
+
+## 常见问题
+
+### 什么是 Umi，为什么需要它？
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)